velox/functions/prestosql/aggregates/DecimalAggregate.h

```diff
@@ -76,10 +93,6 @@ class DecimalAggregate : public exec::Aggregate {
     return sizeof(DecimalAggregate);
   }
 
-  int32_t accumulatorAlignmentSize() const override {
```

Comment:
Did you investigate why this is not working?

Reply:
Yes, we looked into it, I updated the comment accordingly.

Reply:
Should we remove this method from parent class altogether if it is not working as expected?

Reply:
The root cause is that we cannot make the actual address of accumulator aligned at 16 bytes, because this method only affects the offset but not the group address.  So this method is not doing what it is supposed to do and should be removed.

Reply:
I don't understand your concern, this method is removed from DecimalAggregate where it has no effect.

Reply:
For decimal sum and avg this fix is good.  I am more from the perspective of the whole aggregate function framework, since this reveals that relying on this method is a pitfall.  We should address this at framework level, maybe we can do it in a separate PR.  I will raise this to our team.

Reply:
I see what you mean. Yes, this should be done separately as it would involve unrelated to the fix changes.